src/components/Header.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logoCash from './assets/logoCash.png';
+import './Header.css';
+
+const Header = () => (
+  <header className="header">
+    <img className="cashiswineImg" src={logoCash} alt="logo" />
+    <div className="thefourth">
+      <nav className="navlist">
+        <NavLink className="link" activeClassName="active" exact to="/">
+          Accueil
+        </NavLink>
+
+        <NavLink className="link" activeClassName="active" exact to="/wine-library">
+          Vinothèque
+        </NavLink>
+
+        <NavLink className="link" activeClassName="active" exact to="/profile">
+          Compte
+        </NavLink>
```

Comment:
be DRY (don't repeat yourself). take a look here: https://github.com/rocambille/first-login/blob/main/src/components/Navbar.jsx#L6 ;)

Reply:
you should end up with something like that:

    { link('/', 'Accueil') }
    { link('/wine-library', 'Vinothèque') }
    { link('/profile', 'Compte') }